gtests/net/mptcp/common/multi-ep.sh

```diff
@@ -69,19 +69,19 @@ else # ipv4 or ipv4-mapped-ipv6
 EOF
         for o in $d $c $b $a; do
```

Comment:
indeed better to avoid overriding `b` but was it really causing an issue? I guess `$b` was evaluated when building the `for`-loop and then not using a value assigned below, no?

Anyway, clearer with the new version :)

Reply:
> indeed better to avoid overriding `b` but was it really causing an issue? I guess `$b` was evaluated when building the `for`-loop and then not using a value assigned below, no?

right, I just got confused while reading the script and decided to fix all strange stuff in the netmask computing. FTR, this should allow doing something like

{{{
#  ./multi-ep.sh -n 1 -m subflow -b
}}}

and correctly see the 'subflow' endpoint added




> 
> Anyway, clearer with the new version :)